accounts/mixins.py

```diff
@@ -0,0 +1,17 @@
+from django.contrib.auth.mixins import UserPassesTestMixin
+from django.core.exceptions import PermissionDenied
+from django.shortcuts import redirect
+
+
+class AdminRequiredMixin(UserPassesTestMixin):
+    def test_func(self):
+        # Allow access to superusers only
+        return self.request.user.is_superuser
```

Comment:
Super user will be the overall system's super user. But as acc to our ERD there can be multiple Admins each associated to one organization then the admin shouldn't be a super user.